datasets/ONS-foreign-direct-investment-involving-uk-companies/main.py

```diff
@@ -265,24 +273,36 @@ def split_overrides(bag, splits):
     "gulf-arabian": "gulf-arabian-countries",
     "other-asian": "other-asian-countries"
 }
-observations["FDI Area"] = observations["FDI Area"].map(lambda x: fix.get(x, x))
+observations['FDI Area'] = observations['FDI Area'].map(lambda x: fix.get(x, x))
 
 
 # +
-
-inward_scraper.dataset.title = inward_scraper.dataset.title.replace(': inward', '')
-inward_scraper.dataset.comment = inward_scraper.dataset.comment.replace(
-    'into the UK', 'into the UK and of UK companies abroad')
-inward_scraper.dataset.landingPage = landingPages
-
-from gssutils.metadata import THEME
-inward_scraper.dataset.theme = THEME['business-industry-trade-energy']
-inward_scraper.dataset.family = 'trade'
-
-cubes.add_cube(inward_scraper, observations.drop_duplicates(), inward_scraper.dataset.title)
-cubes.output_all()
-
-# -
-trace.render("spec_v1.html")
-
-
+observations.to_csv('foreign_direct_investment-observations.csv', index = False)
+
+catalog_metadata: CatalogMetadata = CatalogMetadata(
+    title = "Foreign direct investment involving UK companies (directional): inward",
+    summary = "Annual statistics on the investment of foreign companies into the UK, including for investment flows, positions and earnings.",
+    description = "The sum of constituent items may not always agree exactly with the totals shown due to rounding. \
```

Comment:
Adding the footnotes in the description here is correct, though need to check there isn't any additional information needed at the beginning of the description that is being overwritten.

Reply:
Added the information which already existed in catalogue description before being overwritten. Catalogue description is now 
```
description = "Inward and Outward reference table including data for flows, positions and earnings. \
The sum of constituent items may not always agree exactly with the totals shown due to rounding. \
A negative sign before values indicates a net disinvestment in the UK. \
Component breakdown excludes the activities of private property, public corporations and bank holding companies.These are included in the total."
```